pkg/audit/verifier.go

```diff
@@ -367,35 +372,58 @@ func (verifier *Verifier) Reverify(ctx context.Context, stripe *Stripe) (report
 					verifier.log.Debug("Reverify: unknown transport error (contained)", zap.Stringer("Node ID", piece.NodeId), zap.Error(err))
 					return
 				}
-
 				if errs2.IsRPC(err, codes.NotFound) {
+					if len(pending.Path) > 0 {
```

Comment:
Since this chunk is repeated, would it make sense to pull it out into a private function?

Reply:
the calls to checkIfSegmentDeleted are slightly different unfortunately